package.json

```diff
@@ -11,8 +11,8 @@
 	},
 	"devDependencies": {
 		"@changesets/cli": "2.27.1",
-		"@guardian/eslint-config": "workspace:*",
-		"@guardian/eslint-config-typescript": "workspace:*",
```

Comment:
was this the only place this was used?

Reply:
yeah, i t made it available to everything in the repo